tutorials/contour/index.pug

```diff
@@ -0,0 +1,69 @@
+extends ../common/index.pug
+
+block mainTutorial
+  :markdown-it
+    # Tutorial - Contour map
+    First, let's create our map and add a base map and a feature layer.
+  
+  +codeblock('javascript', 1).
+    var map = geo.map({
+      node: '#map',
+      center: { x: -157.965, y: 21.482 },
+      zoom: 10
+    });
+    map.createLayer('osm');
+    var layer = map.createLayer('feature');
+
+  :markdown-it
+    Then load some data. 
+    The data is grid data describing geospatial point elevation, a record of z value -9999 means it's sea level. 
```

Comment:
Pedantically, the value of -9999 means that there is no land data (rather than that it is sea level).